media/Config/importConfig.js

```diff
@@ -14,47 +14,63 @@
  * limitations under the License.
  */
 
+/**
+ * @brief oneToolToggle will toggle one-import-xxx and one-xxx of webview.
+ * @param data array that holds all options and values of one-build or onecc
+ * @param importOpt option name of one-build or onecc
+ */
+const oneToolToggle = function(data, importOpt) {
+  for (let i = 0; i < oneToolList.length; i++) {
+    if (data[importOpt] === 'True') {
+      if (importOpt === oneToolList[i].type && i === 0) {
+        oneImport.use = true;
+        chooseImportOption(0);
+      } else if (importOpt === oneToolList[i].type && i === 1) {
+        oneImport.use = true;
+        chooseImportOption(1);
+      } else if (importOpt === oneToolList[i].type && i === 2) {
+        oneImport.use = true;
+        chooseImportOption(2);
+      } else if (importOpt === oneToolList[i].type && i === 3) {
+        oneImport.use = true;
+        chooseImportOption(3);
+      } else if (importOpt === oneToolList[i].type && i > 3) {
+        oneToolList[i].use = true;
+      }
+    }
+  }
```

Comment:
Is this something like this?
```js
  if (data[importOpt] === 'True') {
    for (let i = 0; i < 3; i++) {
      if (importOpt === oneToolList[i].type) {
        oneImport.use = true;
        chooseImportOption(i);
      }
    }
    for (let i = 3; i < oneToolList.length; i++) {
      if (importOpt === oneToolList[i].type) {
        oneImport.use = true;
      }
    }
  }
```
Q) what or why is `3` ?

Reply:
`if (data[importOpt] === 'True') {` line is invariant with `i`, so I think this canbe moved outside of `i` loop.

Reply:
yes. I will update importConfig.js like that. Thank you! and oneToolList is array that has tool names. index 0 to 2 is one-import tool name and the rest are other tool names like optimize, quantize ...

Reply:
>  index 0 to 2 is one-import tool name

If these numbers are fixed and used as some constant numbers, could you please defined them as global varaibles for better understanding the code?